docs/sections/cn_workflows.md

```diff
@@ -417,64 +417,49 @@ by default.
 ### Kubernetes
 
 Popper enables leveraging the compute and storage capabilities of the cloud by allowing running workflows on 
-Kubernetes clusters. One need to have access to the config file and ensure that any PersistentVolume is available
-inside the cluster. Popper takes care of the workflow execution from there.
+kubernetes clusters. Users just need to have access to the cluster config file in order to run workflows on kubernetes.
+Popper provisions all the required resources and orchestrates the entire the workflow execution. The Popper kubernetes
+runner currently supports only the Docker container runtime.
 
-When a workflow is executed, Popper first creates a persistent volume claim and spawns an init pod and uses it to copy the workflow context (compressed in the form of a `.tar.gz` file) into the persistent volume. Then, popper teardowns this init pod and execute the steps of a workflow in separate pods of their own. After the execution of each step, the respective pods are deleted but the persistent volume claim is not deleted, so that it can reused by subsequent workflow executions.
+When a workflow is executed, Popper first creates a persistent volume claim, spawns an init pod and uses it to copy the workflow context (packed in the form of a `.tar.gz` file) into the persistent volume and then unpacks the context there. Then, popper teardowns the init pod and execute the steps of a workflow in separate pods of their own. After the execution of each step, the respective pods are deleted but the persistent volume claim is not deleted, so that it can reused by subsequent workflow executions.
 
-For running workflows on Kubernetes, some configuration options need to be passed to the kubernetes resource manager through the popper configuration file.
+For running workflows on kubernetes, several configuration options can be passed to the kubernetes resource manager through the popper configuration file to customize the execution environment. All the available configuration options has been described below.
 
-#### Running on a cluster without shared storage
+* `namespace`: The namespace within which to provision resources like PVCs and Pods for a workflow execution. If not provided the       `default` namespace will be used.
 
-**NOTE:** If your workflow needs to build an image from a `Dockerfile`, make sure you are logged in to dockerhub.
+* `persistent_volume_name`: Any pre-provisioned persistent volume like an NFS or EBS volume can be supplied through this option. Popper will then claim storage space from the supplied persistent volume. In the default case, a local persistent volume of 1Gi will be created by Popper automatically.
```

Comment:
1GB is more common than 1Gi

Reply:
is there a default name ?

Reply:
So, when volume name is not provided a volume with a name similar to `pv-hostpath-popper-a3rf5fy` is created automatically.

Reply:
Should we mention the default name ?